src/main/java/swettdg/com/CodeFellowship/models/ApplicationUser.java

```diff
@@ -21,6 +21,9 @@
     String dob;
     String bio;
 
+    @ManyToMany

```

Comment:
It looks like you deleted one of your Sets here, and this annotation is preventing your app from running.